src/recommender.py

```diff
@@ -78,6 +78,7 @@
                    is_quickstart_majority, execute_gremlin_dsl)
 from stack_aggregator import extract_user_stack_package_licenses
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
Do we really want this granular logging? Or is it temporary?

Reply:
i have purposefully added it. Otherwise it becomes very difficult to see logs to deduce whats the problem.